css/nav.css

```diff
@@ -177,6 +177,8 @@ nav a#nav-credits:focus-visible:not(.active)::before {
   /* portrait / small width styling, show burger menu */
   #nav-hamburger {
     display: block;
+    /* ensure that this always stays up top */
+    z-index: 999;
```

Comment:
will this show above the dialog then?

Reply:
Ah shoot, you're right. I will need to adjust the dialog z-index also so it's greater than both the hamburger button, which has to be greater than the max z-index value of the last flipped card in the gacha page.